tests/python/contrib/test_hexagon/topi/test_adaptive_avg_pool1d.py

```diff
@@ -0,0 +1,181 @@
+# Licensed to the Apache Software Foundation (ASF) under one

```

Comment:
I have added lint for tests in this sub-directory. Let's wait for that PR to merge to avoid conflict. After that merges, you will see lint issues with this test, so I recommend to fix it in the meantime
https://github.com/apache/tvm/pull/13271